test/e2e/ginkgo-node-1.log

```diff
@@ -0,0 +1,41 @@
+Running Suite: Libpod Suite
```

Comment:
Please remove this log and the other .log files.

Reply:
Just removed these. I also added a `.gitignore` rule for them